battery

```diff
@@ -34,34 +29,31 @@ setDefaults() {
     good_color="1;32"
     middle_color="1;33"
     warn_color="0;31"
-    connected=0
     battery_path=/sys/class/power_supply/BAT0
 }
 
-setDefaults
-
 # Determine battery charge state
 battery_charge() {
-    case $(uname -s) in
+    case "$(uname -s)" in
         "Darwin")
             if ((pmset_on)) && command -v pmset &>/dev/null; then
-                if [ "$(pmset -g batt | grep -o 'AC Power')" ]; then
+                if [[ "$(pmset -g batt | grep -o 'AC Power')" ]]; then
                     BATT_CONNECTED=1
                 else
                     BATT_CONNECTED=0
                 fi
                 BATT_PCT=$(pmset -g batt | grep -o '[0-9]*%' | tr -d %)
             else
                 while read key value; do
-                    case $key in
+                    case "$key" in
```

Comment:
Just wondering why you want to be working with strings here?

Reply:
You're not wrong. The use of quoting is a style I've taken up. I've got bitten by word splitting issues and empty values too many times before - it's simply _safer_ to always quote the variables. But yes, with `case` it's not required.